docs/security-model.mdx

```diff
@@ -0,0 +1,55 @@
+---
+id: security-model
+title: The Ory Security Model
+sidebar_label: Ory Security Model
+---
+
+Ory has a proven security model which can take a moment to get used to. Compared to other solutions, Ory manages sessions, tokens,
+and cookies for you whenever possible. You do not need to worry about refreshing tokens or figuring out whether to store them in
+`localStorage` or `document.cookies`. Instead, Ory solves all of this for you and deploys best-in-class protections for common
+browser attack vectors (XSS, CSRF, ...).
+
+## Ory's Identity Service Need Access to Your Domain
+
+Ory bears the responsibility of managing sessions, cookies, and tokens for you. To do this, Ory needs to be exposed on your
+domain, otherwise HTTP Cookies will not be sent. Per default, Ory's APIs are available at
+
+```
+https://<project-slug>.projects.oryapis.com
+```
+
+Let's say you are developing an app and it runs on `http://localhost:3000`. For Ory to be able to manage the session information,
+it needs to be able to set an HTTP Cookie for `domain=localhost` as well! The same applies when your app runs in production (for
+example on `https://app.example.org`). Here too Ory needs to be able to set the HTTP Cookie for `domain=example.org`:
+
+1. During **development** you use the [Ory Tunnel on your machine](guides/local-development.mdx) to "mirror" the APIs available at
+   `https://<project-slug>.projects.oryapis.com` on your `localhost` domain. Ory offers SDKs for certain deployment options such
+   as [Vercel](guides/protect-page-login/nextjs.mdx) which mirror Ory's APIs without the need of running another process.
+2. For **production** you set up a [Custom Domain](guides/custom-domains.mdx) (for example `https://ory.example.org`) which makes
+   Ory's APIs available at `https://ory.example.org`.
+
+If you're interested in learning more about this topic, head over to
+[HTTP Cookies Explained](further-reading/http-cookies-explained.mdx)
+
+## Frequently Asked Questions
+
+Below we have collected some frequently asked questions about Ory's security model.
+
+### How do I solve CSRF Issues?
```

Comment:
What about this one? Still a WIP or do we need to write something for this paragraph?